CreateStore/bugfix_sdk_1682/Microsoft.NET.ComposeStore.targets

```diff
@@ -0,0 +1,461 @@
+    </PropertyGroup>
+  </Target>
+
+  <!--
+    ============================================================
+                                        _GetResolvedFilesToStore
+    ============================================================
+    -->
+  
+  <Target Name="_GetResolvedFilesToStore"
+           Condition="$(SkipOptimization) == 'true' ">
+    <ItemGroup>
+            <_UnOptimizedResolvedFileToPublish Include="@(ResolvedFileToPublish)" />
+    </ItemGroup>
+  </Target>
+
+  <!--
+    ============================================================
+                                        ComputeFilesToStore
+
+    Gathers all the files that need to be copied to the publish directory.
+    ============================================================
+    -->
+  <UsingTask TaskName="FilterResolvedFiles" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  <Target Name="ComputeFilesToStore"
+          DependsOnTargets="_ComputeNetPublishAssets;
+                            _ComputeCopyToPublishDirectoryItems">
+
+    <PropertyGroup>
+      <CopyBuildOutputToPublishDirectory Condition="'$(CopyBuildOutputToPublishDirectory)'==''">true</CopyBuildOutputToPublishDirectory>
+      <CopyOutputSymbolsToPublishDirectory Condition="'$(CopyOutputSymbolsToPublishDirectory)'==''">true</CopyOutputSymbolsToPublishDirectory>
+    </PropertyGroup>
+
+    <FilterResolvedFiles  AssetsFilePath="$(ProjectAssetsFile)"
+                           ResolvedFiles ="@(ResolvedAssembliesToPublish)"
+                           PackagesToPrune="$(PackagesToPrune)"
+                           TargetFramework="$(TargetFrameworkMoniker)"
+                           RuntimeIdentifier="$(RuntimeIdentifier)"
+                           IsSelfContained="$(SelfContained)" >
+      <Output TaskParameter="AssembliesToPublish" ItemName="ResolvedFileToPublish" />
+      <Output TaskParameter="PublishedPackges" ItemName="PackagesThatWereResolved" />
+    </FilterResolvedFiles>
+
+    <ItemGroup>
+      <ResolvedPackagesPublished Include="@(PackagesThatWereResolved)"
+                                    Condition="$(DoNotTrackPackageAsResolved) !='true'"/>
+    </ItemGroup>
+    
+  </Target>
+
+  <!--
+    ============================================================
+                                       PrepRestoreForStoreProjects 
+
+    Removes specified PackageReference for store and inserts the specified StorePackageName
+    ============================================================
+    -->
+  <Target Name="PrepRestoreForStoreProjects"
+          BeforeTargets="_GenerateProjectRestoreGraphPerFramework;"
+          Condition="'$(StorePackageName)' != ''" >
+    
+    <ItemGroup>
+      <PackageReference Remove="@(PackageReference)" Condition="'%(PackageReference.IsImplicitlyDefined)' != 'true'"/>
+      <PackageReference Include="$(StorePackageName)" Version="$(StorePackageVersion)"/>
+    </ItemGroup>
+  </Target>
```

Comment:
If the fix for the crossgen issues is to make a copy of all of the targets, I think that's OK as a temporary solution until we can get a fixed build. I don't want to diverge too far from the shipping product.

Reply:
I agree I don't want to diverge any farther than necessary... at the moment the difference is my ~4 line fix.
Copying the file was a cheap hack so that I could edit those lines. Since the SDK was now pinned to 2.0.0 I could be guaranteed that was the only difference between the two files. If folks want to allow floating the SDK forward then it either requires taking a risk that the files will diverge further or making a more elaborate scheme to patch the file without copying all of it.